### DISCUSSION TITLE
LangChain or Plugin Support

### DISCUSSION THREAD

**Discussion by gunpal5** [vectorch-ai/ScaleLLM#27] [Q&A]
I am interested to use ScaleLLM for deployment of newly launched 01-AI\Yi models on servers. does ScaleLLM support plugin/function architecture or LangChain like OpenAI?

**Comment by guocuimi** [comment]
Thank you for your interest in ScaleLLM. As of now, we do not yet support plugin or functional calls. However, integrating a search agent is on our roadmap. We plan to prioritize this feature in Q2 of 2024, following the completion of our current focus on performance optimization enhancements. stay tuned!
